content/en/docs/next/quickstart.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Quickstart
+weight: 900
+description: Get etcd up and running in less than 5 minutes!
+---
+
+## Step 1 - Install etcd
+
+Set the environment variables to help with the commands that follow.
+
+```
+ETCD_VER={{< param git_version_tag >}}
+
+GOOGLE_URL=https://storage.googleapis.com/etcd
+GITHUB_URL=https://github.com/etcd-io/etcd/releases/download
+```
+
+Download and install etcd:
+
+On Linux
+```
+$ DOWNLOAD_URL=${GITHUB_URL}
+
+$ rm -f /tmp/etcd-${ETCD_VER}-linux-amd64.tar.gz
+$ rm -rf /tmp/test-etcd && mkdir -p /tmp/test-etcd
```

Comment:
Put the `mkdir` command on its own line (and drop the `&&`).

Do the same think for macOS.